src/ecs/util/resources.rs

```diff
@@ -0,0 +1,45 @@
+//! Util Resources
+use util::circular_buffer::CircularBuffer;
+
+/// The FPSCounter resource needed by the FPSCounterSystem
+/// # Examples
+/// Add it to your resources to be able to use the FPSCounterSystem
+/// ~~~no_run
+/// world.add_resource(FPSCounter::new(20));
+/// ~~~
+/// Usage example:
+/// ~~~no_run
+/// Get the FPSCounter resource from the System or from the world.
```

Comment:
This won't compile; please use a comment and create a counter; also don't forget to import the items you're using.

Reply:
Fixed, its just a comment and not and actual example, as I can't get a world instance without creating a amethyst context.